### PR TITLE
Adding echidna workflow

### PR DESCRIPTION
**Pull request by iherman**
Title tells it all

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: .github/workflows/auto-publish.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - uses: w3c/spec-prod@v2
+        with:
+          TOOLCHAIN: respec
+          W3C_ECHIDNA_TOKEN: ${{ secrets.W3C_TR_TOKEN }}
+          W3C_WG_DECISION_URL: https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-04-12-vcwg#resolution1
+          W3C_BUILD_OVERRIDE: |
+             shortName: vc-jwt
+             specStatus: WD
```
Comment (OR13) [inline .github/workflows/auto-publish.yml:20]: do i need to update the index.html to this as well?

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
I am happy to merge this as is, and then correct the errors that are causing CI to fail.

File: .github/workflows/auto-publish.yml (hunk 1 above)
Comment (iherman) [reply .github/workflows/auto-publish.yml:20]: You mean setting the specStatus to WD? No. Echidna, more exactly the respec engine does that for you (it is possible to start up the respec transform engine with query parameters in the URL to overwrite virtually any respec configuration value).

**Review by selfissued — COMMENTED** [review]
I think we should proceed with this.

**Comment by OR13** [comment]
@iherman thank you!

it is working now:

https://www.w3.org/TR/vc-jwt/
Screen Shot 2023-05-01 at 3 50 48 PM

**Pull request merged**